### DISCUSSION TITLE
Tier Telemetry Data

### DISCUSSION THREAD

**Discussion by Vucko95** [theOehrly/Fast-F1#422] [Q&A] [answered]
Hi,
 I wonder how are you pulling the data for `Tier Compound per race` , I'm using Ergast copy of DB locally, but I cannot find in the DB any data regarding Tier Compounds.

**Comment by Casper-Guo** [comment]
Do you mean compounds as in [SOFT, MEDIUM, HARD] or [C1, C2...]?

**Comment by Vucko95** [comment]
Yes

**Comment by Vucko95** [comment]
Okay I think I found the site to where Request are made . 
https://livetiming.formula1.com/ 
as I tried for example "https://livetiming.formula1.com/static/2022/2022-07-31_Hungarian_Grand_Prix/2022-07-31_Race/"
Buy I don't have a key, and I'm not sure how to register.
How can I access that page , is this official page for Timing Data. 
As I would need to get that data for race telemetry, to add it to my local database , as Ergast is missing that data.

**Reply by theOehrly** [reply, marked as answer]
Yes, this is the official API and the source for this data in FastF1. You don't need any key to access the API.

Compound information is available through (for example).

https://livetiming.formula1.com/static/2022/2022-07-31_Hungarian_Grand_Prix/2022-07-31_Race/TimingAppData.json

This endpoint will give you an overview over the available topics: https://livetiming.formula1.com/static/2022/2022-07-31_Hungarian_Grand_Prix/2022-07-31_Race/Index.json

I'd recommend to just pull the data using FastF1, so you don't need to parse it yourself, though. Compound information is not the most complicated thing but you should expect weird edge cases. 

Edit: you can also take a look at this comment for more info https://github.com/theOehrly/Fast-F1/issues/248#issue-1355859535

**Reply by Vucko95** [reply]
Thank you this will help a lot !
